package.json

```diff
@@ -12,7 +14,9 @@
     "redux": "^4.0.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --output-style compressed",
+    "watch-css": "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive --skip-initial",
+    "start": "react-scripts watch-css && react-scripts start",
```

Comment:
If you feel adventurous someday try to eject and add a [sass loader](https://github.com/webpack-contrib/sass-loader) 😝

Reply:
@papplo